src/forwarder.ts

```diff
@@ -28,12 +39,25 @@ interface DescribeSubscriptionFiltersResponse {
 function isLogGroup(value: any): value is LogGroupResource {
   return value.Type === logGroupKey;
 }
+/**
+ * Validates whether ARN exists in the account
+ * @param aws Serverless framework provided AWS client
+ * @param functionArn The ARN to be validated
+ */
+async function doesArnExist(aws: Aws, functionArn: string) {
+  try {
+    const result = await aws.request("Lambda", "getFunction", { FunctionName: functionArn });
+  } catch (err) {
+    throw new DatadogForwarderNotFoundError(`Could not perform GetFunction on ${functionArn}.`);
```

Comment:
I would not throw `DatadogForwarderNotFoundError` in a function that checks for the existence of a Lambda that may or may not be the Forwarder. What if we called this function on a non-Forwarder lambda? If we didn't find it, we wouldn't want the error to say anything about the Forwarder.

A couple of ideas:
- You could rename this function so that it is clear that it's specifically looking for a Forwarder, not any old Lambda.
- You could leave the function more general, but then move the error handling code outside of this function into a context where we know we're talking about the Forwarder.